glean-core/src/upload/directory.rs

```diff
@@ -76,41 +95,49 @@ impl PingDirectoryManager {
     pub fn new<P: Into<PathBuf>>(data_path: P) -> Self {
         let data_path = data_path.into();
         Self {
-            pings_dirs: [
-                data_path.join(PENDING_PINGS_DIRECTORY),
-                data_path.join(DELETION_REQUEST_PINGS_DIRECTORY),
-            ],
+            pending_pings_dir: data_path.join(PENDING_PINGS_DIRECTORY),
+            deletion_request_pings_dir: data_path.join(DELETION_REQUEST_PINGS_DIRECTORY),
         }
     }
 
     /// Attempts to delete a ping file.
     ///
-    /// ## Arguments
+    /// # Arguments
```

Comment:
We're _nearly_ consistent in using a level-2 header (`##`), which seems to be the exact opposite of what the Rust libstd does.
It's also noted in the [API guidelines](https://rust-lang.github.io/api-guidelines/documentation.html) (an old thing that's not fully maintained anymore I think) and this RFC uses level-1 header: [RFC 1574](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md).

Where am I going with this? I initially intended to argue for keeping level-2, but I guess it's better if I file a bug and we streamline this to level-1 eventually.

Please move on, nothing to see here.

Reply:
I'll be happy to take that bug, when you file it. I did not know about this official guidelines, I just get annoyed everytime I realize this stuff is not standardized and try to do just that.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1657112